src/containers/ClassifierContainer.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+//import { checkLoginUser, loginToPanoptes, logoutFromPanoptes } from '../ducks/login';
```

Comment:
I take it these are the ducks that still need to be written for classifier functionality.